src/main/java/com/coveo/spillway/storage/LimitUsageStorage.java

```diff
@@ -120,6 +120,28 @@
   Map<LimitKey, Integer> debugCurrentLimitCounters();
 
   /**
+   * @param resource The resource for which you want to get the current limit counts
+   * @return A Map of all the enforced limits for a resource and their current count
+   *
+   */
+  Map<LimitKey, Integer> getCurrentLimitCounters(String resource);
```

Comment:
You'd need to store all the keys or properties associated with a resource in a duplicated storage somewhere to be able to retrieve your limits for a resource. You'd also have to do a call for every individual limits which seems really inefficient.